### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by n1k0**
r=? @almet @leplatrem @natim

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -103,20 +132,17 @@
 
 tasks
   .create({label: "First item", done: false})
-  .then(_ => tasks.sync())
-  .then(_ => tasks.list())
-  .then(records => { console.log(records)});</code>
-        </pre>
+  .then(_ => tasks.sync());</code></pre>
```
Comment (almet) [inline index.html]: That actually was pretty cool to see the tasks.list in action, I think.

File: index.html [hunk 2]
```diff
@@ -103,20 +132,17 @@
...
           <h2>Ready for the browser</h2>
           <p><a href="http://kintojs.readthedocs.org/">Kinto.js</a> helps you to build Web Apps that syncs.</p>
           <ul>
-            <li>Unlimited local space with IndexedDB</li>
-            <li>Keep working when offline</li>
-            <li>Transparent sync</li>
-            <li>Built in conflict resolutions strategies</li>
+            <li><a href="http://offlinefirst.org/">Offline first</a></li>
+            <li>In-browser storage using IndexedDB</li>
+            <li>Transparent synchronisation flow</li>
```
Comment (almet) [inline index.html]: z or s ?

Comment (n1k0) [inline index.html]: z is US, s is UK and rest of the world. Your call.

File: index.html (hunk 1 above)
Comment (n1k0) [inline index.html]: I like it more focused on the feature highlighted, but I'm not against showcasing list() really. Your call.

Comment (leplatrem) [inline index.html]: I also vote for a shorter piece of code :)

**Pull request merged**